app/main.py

```diff
@@ -1,6 +1,38 @@
+import os
+
+from dotenv import load_dotenv
+
+import requests
+
+
+load_dotenv()
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    api_key = os.getenv("API_KEY")
+    if not api_key:
+        raise ValueError("API_KEY environment variable is not set")
+
+    city = "Paris"
+    base_url = "http://api.weatherapi.com/v1/current.json"
```

Comment:
According to the checklist, constants should be defined in uppercase. Consider defining `city` and `base_url` as constants in uppercase, like `CITY = "Paris"` and `BASE_URL = "http://api.weatherapi.com/v1/current.json"`.